lib/tasks/delete_routes.rake

```diff
@@ -0,0 +1,28 @@
+	collections = ["/government/collections/ad-hoc-statistical-analysis-2015-quarter-1", "/government/collections/cde-marketplace-5-february-2015-exhibitor-case-studies", "/government/collections/chapter-32-port-cases-involving-prosecution-immigration-directorate-instructions", "/government/collections/command-papers", "/government/collections/departmental-exceptions-to-spending-controls-2014", "/government/collections/electronic-business-commissioners-directions", "/government/collections/flagging-up-newsletters", "/government/collections/green-deal-quick-guides", "/government/collections/greenhouse-gas-conversion-factors-for-company-reporting", "/government/collections/guidance-on-british-citizenship", "/government/collections/house-of-commons-papers", "/government/collections/ministerial-gifts-hospitality-travel-and-meetings-2012", "/government/collections/ministerial-gifts-hospitality-travel-and-meetings-2013", "/government/collections/national-curriculum-assessments-2013", "/government/collections/official-documents", "/government/collections/oisc-news", "/government/collections/self-assessment-helpsheets-additional-information", "/government/collections/social-care-online-questionnaires-2015", "/government/collections/think-act-report-sign-ups-and-case-studies"]
+
+	case args[:paths_type]
+	when 'html_attachment_ids'
+		regex = /\/government\/publications\/[a-zA-Z0-9\-]\/(#{html_attachment_ids.join("|")})/
+		routes = Route.where(incoming_path: regex, backend_id: "whitehall-frontend")
+	when 'collections'
+		routes = Route.where(incoming_path: collections, backend_id: "whitehall-frontend")
+	when 'all'
+		routes = Route.where(incoming_path: [regex, collections].flatten, backend_id: "whitehall-frontend")
```

Comment:
From looking at the code, I don't think the value `regex` here will be defined. Has this path been tested?